runners/core-java/src/main/java/org/apache/beam/runners/core/SimpleDoFnRunner.java

```diff
@@ -213,19 +200,19 @@ public void finishBundle() {
     }
   }
 
-  private DoFnStartBundleContext<InputT, OutputT> createStartBundleContext(
+  private DoFnStartBundleContext createStartBundleContext(
       DoFn<InputT, OutputT> fn, DoFnContext<InputT, OutputT> context) {
```

Comment:
Did more cleanup by making all of the context classes non-static and eliminating unnecessary member variables and functions. DoFnContext went away completely.